addons/portal_rating/static/src/js/portal_composer.js

```diff
@@ -59,19 +59,43 @@ PortalComposer.include({
         var self = this;
         return this._super.apply(this, arguments).then(function () {
             // rating stars
-            self.$input = self.$('input[name="rating_value"]');
+            self.$inputRating = self.$('input[name="rating_value"]');
             self.$star_list = self.$('.stars').find('i');
 
             // set the default value to trigger the display of star widget and update the hidden input value.
             self.set("star_value", self.options.default_rating_value); 
-            self.$input.val(self.options.default_rating_value);
+            self.$inputRating.val(self.options.default_rating_value);
         });
     },
 
     //--------------------------------------------------------------------------
     // Handlers
     //--------------------------------------------------------------------------
 
+    /**
+     * @override
+     * @private
+     */
+    _onSubmitButtonClick: async function (ev) {
+        const result = await this._super(...arguments);
+        if (result) {
+            //close rating bootstrap modal
+            this.$el.closest('#ratingpopupcomposer').modal('hide');
+            // need to recalculate rating popup composer value
+            this.trigger_up('reload_rating_popup_composer', result);
+        }
+        return result;
```

Comment:
Same here, this is an event handler, nothing will capture that return value, so this return is useless.